templates/taxi/driver_form.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  {% if form_type == "license" %}
```

Comment:
why do you need this if? you can simply update `yesno:"Update..`

Reply:
The previous version of my code included a driver info update feature, for which it was logical. 
I updated this block for the current version.